images/linux/scripts/installers/pypy.sh

```diff
@@ -75,14 +72,31 @@ function InstallPyPy
 function getPyPyVersions
 {
     uri="https://downloads.python.org/pypy/"
-    wget -q -O - $uri | gunzip -c | grep 'linux64' | awk -v uri="$uri" -F'>|<' '{print uri$5}'
+    i=20
+
+    while [ $i -gt 0 ]; do

```

Comment:
I suggest to implement helper function to download file.
You implement good logic so it will let us reuse it in any other places if needs

Reply:
Let's skip this for now to unblock the rollout. I'll create another issue for that.